internal/command/scale/count_machines.go

```diff
@@ -0,0 +1,283 @@
+package scale
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"strings"
+
+	"github.com/samber/lo"
+	"github.com/superfly/flyctl/api"
+	"github.com/superfly/flyctl/flaps"
+	"github.com/superfly/flyctl/internal/appconfig"
+	"github.com/superfly/flyctl/internal/flag"
+	mach "github.com/superfly/flyctl/internal/machine"
+	"github.com/superfly/flyctl/internal/prompt"
+	"github.com/superfly/flyctl/iostreams"
+)
+
+func runMachinesScaleCount(ctx context.Context, appName string, expectedGroupCounts map[string]int, maxPerRegion int) error {
+	io := iostreams.FromContext(ctx)
+
+	flapsClient, err := flaps.NewFromAppName(ctx, appName)
+	if err != nil {
+		return err
+	}
+	ctx = flaps.NewContext(ctx, flapsClient)
+
+	appConfig, err := appconfig.FromRemoteApp(ctx, appName)
+	if err != nil {
+		return err
+	}
+	ctx = appconfig.WithConfig(ctx, appConfig)
+
+	machines, err := mach.AppV2ListActive(ctx)
+	if err != nil {
+		return err
+	}
+
+	if len(machines) == 0 {
+		fmt.Fprintf(io.Out, "There are not active machines for this app. Did you ever `fly deploy`? if not, do it now\n")
```

Comment:
this almost comes off as kind of aggro to me, perhaps something along the lines of "There are no active machines for this app. Have you run 'fly deploy' on this application yet?"

Reply:
```suggestion
		fmt.Fprintf(io.Out, "There are no active machines for this app. Run `fly deploy` to create one, and then re-run this command.\n")
```

Reply:
oh actually, let's return this message as the error message. It will get printed for the developer and I like it when errors tell developer how to correct them.

Reply:
Thanks for looking into those messages! English is not my first lang and sometimes the tone doesn't work the way I think. I will make the wording changes asap.

Reply:
Fixed by 84ca42c2